### PR TITLE
1898

### PR DESCRIPTION
**Pull request by wojciechtanski**

### CODE REVIEW

**Comment by robotci** [comment]
Admin, can CI be enabled?

File: atest/testdata/standard_libraries/reserved.robot [hunk 1]
```diff
@@ -12,6 +12,15 @@ Reserved in User Keyword
...
+    ELSE    log   log something
+
+Else If should be capitalized
+    [Documentation]     FAIL 'else if' is a reserved keyword. 'else if' must be in uppercase (ELSE IF) when used as a marker with 'Run Keyword If'.
+    ELSE IF    log   log something
+
 *** Keyword ***
 User keyword with reserved keyword
     While
+
```
Comment (pekkaklarck) [inline atest/testdata/standard_libraries/reserved.robot]: Extra empty line added here.

File: src/robot/libraries/Reserved.py [hunk 2]
```diff
@@ -23,4 +23,8 @@ def get_keyword_names(self):
         return RESERVED_KEYWORDS
 
     def run_keyword(self, name, args):
-        raise Exception("'%s' is a reserved keyword" % name.title())
+        error = "'%s' is a reserved keyword." % name
```
Comment (pekkaklarck) [inline src/robot/libraries/Reserved.py]: Should it be `name.title()` here? Do other tests pass without it?

**Comment by pekkaklarck** [comment]
Can you @wojciechtanski take a look at the final line notes? After fixing the issues, please squash commits and this is ready to be merged.

**Comment by pekkaklarck** [comment]
If you aren't familiar with squashing, here are pretty good instructions:
http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

Notice also that you don't typically need to create a new PR if you squash commits or update code otherwise. Many PRs make it harder for us to follow what's going on.

**Comment by wojciechtanski** [comment]
changes pushed, i hope all is fine now.

**Comment by wojciechtanski** [comment]
indeed - some tests were not working, did changes there as well

File: atest/robot/standard_libraries/reserved.robot [hunk 3]
```diff
@@ -13,3 +13,9 @@ Continue
 
 Reserved in User Keyword
     Check testcase    Reserved In User Keyword
+
+Else should be capitalized
+    Check testcase    Else should be capitalized
+
+Else If should be capitalized
+    Check testcase    Else If should be capitalized
```
Comment (pekkaklarck) [inline atest/robot/standard_libraries/reserved.robot]: Again missing a newline at the end of the file.

**Comment by pekkaklarck** [comment]
Looks good now except that one file has an extra newline at the end and another file doesn't have any newline at the end. Yes, we are strict about style issues like this.

After fixing newlines, please squash commits and I'll merge this.

**Comment by pekkaklarck** [comment]
Looks good to be merged. Could you @wojciechtanski still quash commits before that?

**Comment by wojciechtanski** [comment]
all code has been already squashed.

**Comment by pekkaklarck** [comment]
There are still four commits. See here fore instructions:
http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

**Comment by pekkaklarck** [comment]
I squashed the commits myself and committed 118498f6e79b927083ef8b9fbccf0c78bdf08368.

**Pull request closed without merging**